aleph/analysis/extract.py

```diff
@@ -42,13 +44,30 @@ def location_country(location):
         return []
 
 
-def extract_entities(text):
+def get_language(text):
+    """Given a list of lines, return a list of (line, lang)"""
+    if not hasattr(settings, '_lang_detector'):
+        lid_model = FastText()
+        lid_model.load_model(settings.LID_MODEL_PATH)
+        settings._lang_detector = lid_model
+    langs = settings._lang_detector.predict([text])
+    return langs[0]
```

Comment:
Are we sure this array always has a length?

Reply:
Yes, the result when it doesn't detect anything is a list of empty lists; an empty list for each snippet of text passed in.